src/domains/vote/components/DelegateTable/DelegateTable.tsx

```diff
@@ -170,15 +191,20 @@ export const DelegateTable = ({ title, coin, delegates, votes, onContinue }: Del
 												className="text-theme-neutral-900"
 												data-testid="DelegateTable__footer--total"
 											>
-												{selected.length}/1
+												{coin === "ARK" ? "1/1" : votesSelected.length + unvotesSelected.length}
```

Comment:
Hardcoding coins is not allowed, as previously mentioned, as it will get messy to support multiple coins that way. Use the helper methods from the SDK to determine how many votes can be selected.

Reply:
@faustbrian I used the `network.maximumVotes()` to get the value.